### PR TITLE
Changed the show_data function

### PR DESCRIPTION
**Pull request by vedangj044**
Fixes #137 

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

I have changed the show data function in querying_mechanism. Now it outputs similar to the following
`[[['id', 36], ['labels', ['collection']], ['properties', [['type', 'MessageCollection'], ['id','vocab:EntryPoint/MessageCollection'], ['operations', "['GET', 'PUT']"]]]]] [[['id', 37], ['labels', ['collection']], ['properties', [['type', 'CommandCollection'], ['id', 'vocab:EntryPoint/CommandCollection'], ['operations', "['GET', 'PUT']"]]]]] [[['id', 38], ['labels', ['collection']], ['properties', [['type', 'StateCollection'], ['id','vocab:EntryPoint/StateCollection'], ['operations', "['GET', 'PUT']"]]]]]`

`Query internal execution time: 0.903287 milliseconds`

### CODE REVIEW

**Comment by vedangj044** [comment]
Fixes #139 

#### Description
Declaring the variable `endpoint_collection_node` in `collection_endpoint.py` in redis_core fixes the bug

File: hydra_agent/querying_mechanism.py [hunk 1]
```diff
@@ -48,30 +48,16 @@ def show_data(self, get_data):
...
+        all_property_lists = ""
+        for i in get_data:
+            try:
+                if isinstance(i[0], list):
+                    for j in i:
+                        all_property_lists+=(str(j)+"\n")
+                else:
+                    all_property_lists+=(str(i[0])+"\n")
+            except Exception as e:
+                all_property_lists+=(str(i)+"\n")
```
Comment (sandeepsajan0) [inline hydra_agent/querying_mechanism.py]: @vedangj044 have you run tests on this? Also, please use better naming like `objects` which could atleast give an idea about its content. You may have searched for best practices of naming in python.

Comment (vedangj044) [reply hydra_agent/querying_mechanism.py]: Okay👍

**Comment by vedangj044** [comment]
Please review @sandeepsajan0 
Thanks !

**Comment by sandeepsajan0** [comment]
@vedangj044  Can you please run tests and send the screenshots?

**Comment by vedangj044** [comment]
Selection_005
Selection_006

**Comment by xadahiya** [comment]
@Guttz please review

**Comment by sandeepsajan0** [comment]
looking fine for me. @Guttz take a look.

**Comment by vedangj044** [comment]
Please Merge

**Comment by xadahiya** [comment]
The count logic that you removed had some very important purpose I don't remember. @Guttz ??

**Comment by chrizandr** [comment]
Can this be merged? @xadahiya

**Comment by vedangj044** [comment]
@sandeepsajan0 Here are the outputs of `show endpoints` `show classEndpoints ``show collectionEndpoints `and `show StateCollection members` queries.
querying_mechanism py — ~-Downloads-learn5-hydra-python-agent — Atom_008
querying_mechanism py — ~-Downloads-learn5-hydra-python-agent — Atom_010

**Comment by vedangj044** [comment]
@sandeepsajan0 @xadahiya do i need to make any other changes ?

**Pull request merged**